tox.ini

```diff
@@ -18,6 +18,14 @@ deps =
   -r{toxinidir}/docs_guidelines/requirements-extra.txt
 commands = sphinx-build -b html -W {posargs} docs_guidelines/ docs_guidelines/_build/html
 
+[testenv:doctest]
+usedevelop = True
+install_command = pip install -U {opts} {packages}
+deps =
+  -r{toxinidir}/requirements-dev.txt
+  -r docs_guidelines/requirements-extra.txt
```

Comment:
Should we use `toxinidir` here too?

`-r{toxinidir}/docs_guidelines/requirements-extra.txt`

Reply:
Good catch!